packages/shared_preferences/shared_preferences_linux/example/lib/main.dart

```diff
@@ -24,22 +24,22 @@ class MyApp extends StatelessWidget {
 }
 
 class SharedPreferencesDemo extends StatefulWidget {
-  SharedPreferencesDemo({Key key}) : super(key: key);
+  SharedPreferencesDemo({Key? key}) : super(key: key);
 
   @override
   SharedPreferencesDemoState createState() => SharedPreferencesDemoState();
 }
 
 class SharedPreferencesDemoState extends State<SharedPreferencesDemo> {
   final prefs = SharedPreferencesLinux.instance;
-  Future<int> _counter;
+  Future<int>? _counter;
```

Comment:
nits: Maybe this should be `late Future<int> _counter'`?

Reply:
Fixed in all examples. I think I was confusing myself by thinking of it as being initialized asynchronously so needing to be `?`, forgetting that the *value* is async, but the future itself isn't.